scheduler/job/job.go

```diff
@@ -297,3 +303,118 @@ func (j *job) syncPeers() (string, error) {
 
 	return internaljob.MarshalResponse(hosts)
 }
+
+// listTasks is a job to list tasks.
+func (j *job) listTasks(ctx context.Context, data string) (string, error) {
+	ctx, cancel := context.WithTimeout(ctx, listTasksTimeout)
```

Comment:
I didn't see any reference or usage for the `ctx` in this method.

Reply:
Ok, this file should belong to another scheduler PR, which I have currently removed.